### PR TITLE
96/check player turn

### PR DESCRIPTION
**Pull request by Hairash**
#96

### CODE REVIEW

File: backend/src/game_server.rs [hunk 1]
```diff
@@ -155,7 +160,7 @@ impl GameServer {
         }
 
         self.broadcast(&game);
-        self.next_turn();
+        self.send_turn();
```
Comment (bronvic) [inline backend/src/game_server.rs:163]: first: next turn changes player and send turn. I can't see where player changes now

second: remove next_turn if you don't use it anymore

File: backend/src/game_server.rs [hunk 2]
```diff
@@ -29,9 +35,12 @@ impl Actor for GameServer {
 impl Handler<inner::Request<Move>> for GameServer {
     type Result = ();
 
-    #[instrument(skip(self))]
     fn handle(&mut self, message: inner::Request<Move>, _: &mut Self::Context) -> Self::Result {
-        debug!("Appstate process move");
+        debug!("Handle move");
+        if !self.check_player_turn(&message.sender) {
```
Comment (bronvic) [inline backend/src/game_server.rs:40]: You should try to use this error: they are awesome!

**Review by bronvic — CHANGES_REQUESTED** [review]
- You should rebase or smth because you have commit with hex highlighting here
- Rustfmt failed
- I don't see where player changes in the end of the turn now
- And there are no tests or pretty errors - but this are not blockers for PR

File: backend/src/game_server.rs (hunk 1 above)
Comment (Hairash) [reply backend/src/game_server.rs:163]: Answer to both questions:
https://github.com/Miol-Mor/battle-game/blob/d08aaf474322d3f72cd4c57e0f9abaeb0e11ab5a/backend/src/game_server.rs#L72

Comment (bronvic) [reply backend/src/game_server.rs:163]: I see now!

**Review by bronvic — APPROVED** [review]

**Review by alex-eg — APPROVED** [review]

**Pull request merged**